.codesandbox/tasks.json

```diff
@@ -14,9 +27,6 @@
       "runAtStart": true,
       "preview": {
         "port": 3000
-      },
-      "restartOn": {
-        "files": ["yarn.lock", "packages/app/scripts/*"]
```

Comment:
wow, need to fix this, we have a bug with generating the task config, probably it omits the restartOn

Reply:
Is this something that we need to address somewhere or is it limited to this repository?

Reply:
created a ticket for the editor team, it's a pitcher problem